### PR TITLE
Fix minishift steps

### PR DESCRIPTION
**Pull request by Gl4di4torRr**
# Changes

1. Changed the name of `build.yml` to `build.yaml`
2. Added a config trigger on the build config to kick off a build automatically.
3. Adding some spacing in the README 

This addresses #235

### CODE REVIEW

**Comment by Gl4di4torRr** [comment]
I am cleaning up my git commits...

**Comment by jessm12** [comment]
Thanks for this @Gl4di4torRr! please might you be able to add something here to detail how to access the dashboard once installed via this method? i.e through the defined `route` URL

**Comment by Gl4di4torRr** [comment]
@jessm12 yes, i will add that.

**Comment by jessm12** [comment]
@Gl4di4torRr perfect, thank you. 

One last question I have is did you set up any RBAC permissions when you installed as I am seeing `pipelineruns.tekton.dev is forbidden: User "system:serviceaccount:openshift-pipelines-operator:default" cannot list pipelineruns.tekton.dev at the cluster scope: no RBAC policy matched` are there clusterroles/bindings that you used for example?

File: README.md [hunk 1]
```diff
@@ -34,7 +34,15 @@ The `install-dev.sh` script will build and push an image of the Tekton Dashboard
 3. Install deployment config `$oc process -f config/templates/deploy.yaml | oc apply -f-`
 4. Install build config `$oc process -f config/templates/build.yaml | oc apply -f-`
 
+## Install on Minishift
+
+1. Install [tektoncd-pipeline-operator](https://github.com/openshift/tektoncd-pipeline-operator#deploy-openshift-pipelines-operator-on-minikube-for-testing)
+2. [Checkout](https://github.com/tektoncd/dashboard/blob/master/DEVELOPMENT.md#checkout-your-fork) the repository
+3. Install deployment config `$ oc process -f config/templates/deploy.yaml | oc apply -f-`
+4. Install build config `$ oc process -f build.yaml | oc apply -f-`
```
Comment (steveodonovan) [inline README.md]: Is this meant to point to `config/templates/build.yml` or preferably `config/templates/build.yaml` the extensions for `deploy.yaml` and `build.yml` should probably be consistent

**Comment by CarolynMabbott** [comment]
@Gl4di4torRr Are you still working on this issue? If not then no worries, I am happy to take over

**Comment by Gl4di4torRr** [comment]
I am planning to finish this up. I am currently out of the country with
limited access so it will be another week before I get back. If that's too
long, feel free to pick it up.

File: README.md (hunk 1 above)
Comment (Gl4di4torRr) [reply README.md]: This is meant to point to `build.yaml` to stay consistent.

**Comment by Gl4di4torRr** [comment]
@CarolynMabbott looks like the changes were made and we can close this pr?

**Comment by CarolynMabbott** [comment]
@Gl4di4torRr Yep we can close this PR now

**Comment by Gl4di4torRr** [comment]
@CarolynMabbott sounds good. that's what i get for a 2 week vacation :)

**Pull request closed without merging**